tests/rptest/services/admin.py

```diff
@@ -322,8 +323,12 @@ def as_dict(self):
 class InboundTopic:
     def __init__(self,
                  source_topic_reference: NamespacedTopic,
```

Comment:
`topic_reference` parameter is meant to include optional components: location hint and initial revision. It is used like this in `test_conflicting_names`. If we are splitting it into 3 we should rename it to `topic_name` and add not only `location_hint` but also `topic_revision`.